### PR TITLE
Remove copy of iprange

### PR DESCRIPTION
**Pull request by xiam**
Hey @ArturoVM, this PR removes the local `iprange` copy and uses the new import path. Could you give it a look?

### CODE REVIEW

**Comment by arturovm** [comment]
Sure!

File: Makefile [hunk 1]
```diff
@@ -15,7 +15,7 @@ build: generate vendor-sync
 all: docker-build
 
 generate:
```
Comment (arturovm) [inline Makefile]: This generate step is no longer necessary, since the `iprange` repo has the latest generated code.

**Review by arturovm — CHANGES_REQUESTED** [review]
Looks good to me, except for that one unnecessary step!

**Comment by arturovm** [comment]
Hey @xiam, I removed that generate step, but I don't have push access to this repo, I think.

**Comment by xiam** [comment]
> Hey @xiam, I removed that generate step, but I don't have push access to this repo, I think.

Oops, no. Sorry, fixed now.

**Review by arturovm — APPROVED** [review]

**Pull request merged**